scripts/insert_into_appointments.sql

```diff
@@ -1 +1,10 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(pasurname,given_name,gender,height,weight,born_on) VALUES
+  ('Rollins', 'Henry', 'm', 69, 210, '1961-02-13'),
+  ('Smith', 'Patti', 'f', 68, 110, '1947-12-30');
+
+INSERT INTO appointments(patient_given_name, patient_surname, doctor_given_name, doctor_surname, appt_time, date_of) VALUES
+  ('Marsha', 'Wilcox', 'Dr. Vicky', 'Hilton', '10:00am', '2016-08-12'),
+  ('Ivory', 'Tailey', 'Dr. Spencer', 'Hubbard', '01:00pm', '2016-08-17'),
+  ('Marsha', 'Wilcox', 'Dr. Spencer', 'Hubbard', '10:00am', '2016-08-12'),
+  ('Blake', 'Hodge', 'Dr. Vicky', 'Hilton', '01:00pm', '2016-08-17');
```

Comment:
1) Recall that the 3 tables, doctors, patients, appointments, were created with specific parameters. are you utilizing those _exact_ parameters? 
2) I see that you are writing out the string literals that you want to insert into the appointments table. One thing to remember is that whatever you insert should be of a format that will be searchable. Do we really want Dr. Hubbard's first name to be "Dr. Spencer"? Probably not. Similarly, date/time should be of a value that is of that type, which then can be manipulated and interpreted into different formats. If it's just a string, your users would not necessarily know how to query. 

Here's 1 solution:
```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```